### PR TITLE
Fixes #179 : Travis build failing

### LINKED ISSUE #179: Travis Build failing because of new update
Travis says: 
```
We have updated our Trusty images on Sept. 7th, 2017. 
You can add group: deprecated-2017Q3 in sudo-enabled builds to use the previous version.
More details here.
```

So we have to add trusty statements to travis.yml

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixed the build fail due to Travis update recently!

Thanks @jithinbp for the guide and info!

I've cloned the exisiting pslab-desktop-apps repo and edited travis.yml to test the build. Find it [here](https://travis-ci.org/CloudyPadmal/testPSLAB-Desktop-Apps/jobs/273757008)

### CODE REVIEW

**Comment by CloudyPadmal** [comment]
Python 2.7 version build passing and Python 3.4 version build failing! I am working on the failing part!

File: .travis.yml [hunk 1]
```diff
@@ -53,9 +54,9 @@ install:
     - sudo python setup.py install
     #This had better work. explicitly adding pslab-python to the path variable
 
-    - cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/
-    - cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/
-    - cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.4.4/lib/python3.4/site-packages/     
+    - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python2.7.13/lib/python2.7/site-packages/; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 3.3 ]]; then cp -R /usr/local/lib/python2.7/dist-packages/PSL /home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/; fi
```
Comment (jithinbp) [inline .travis.yml]: In lines 7, 12 only Python versions 2.7 and 3.4 are being used, so 3.3 either needs to be enabled, or this line removed

**Comment by jithinbp** [comment]
@CloudyPadmal 3.4 build is failing because `python --version` returns `Python 3.4.6`, and pslab-python files are being copied to `3.4.4`, a directory which doesn't exist

**Comment by CloudyPadmal** [comment]
Hi @jithinbp ! I've changed the python folder names to `3.4.6` and removed `3.3` check. Now the build is passing. :D Thanks for the support! :1st_place_medal:

**Comment by CloudyPadmal** [comment]
Thanks all! I will merge this so that I can rebuild #178 !! :)

**Pull request merged**